File: mobile_app/app_pedidos_ios/Objects/src/B4iProject/LS_frmcategoria.m

#import "iCore.h"
@interface LS_frmcategoria:NSObject
@end

@implementation LS_frmcategoria

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"panelfondocategoria"].Top = (int)(0.0);
[views get:@"panelfondocategoria"].Height = (int)((100 / 100.0 * height) - (0.0));
[views get:@"panelfondocategoria"].Left = (int)(0.0);
[views get:@"panelfondocategoria"].Width = (int)((100 / 100.0 * width) - (0.0));
[views get:@"panelencabezadocat"].Left = (int)(0.0);
[views get:@"panelencabezadocat"].Width = (int)((100 / 100.0 * width) - (0.0));
[views get:@"panelencabezadocat"].Top = (int)(0.0);
[views get:@"panelencabezadocat"].Height = (int)((8 / 100.0 * height) - (0.0));
[views get:@"btnbackcat"].Top = (int)(0.0);
[views get:@"btnbackcat"].Height = (int)((8 / 100.0 * height) - (0.0));
[views get:@"btnbackcat"].Left = (int)(0.0);
[views get:@"btnbackcat"].Width = (int)((15 / 100.0 * width) - (0.0));
[views get:@"label1"].Left = (int)(([views get:@"btnbackcat"].Left + [views get:@"btnbackcat"].Width)+(4 / 100.0 * width));
[views get:@"label1"].Top = (int)(0.0);
[views get:@"label1"].Height = (int)((8 / 100.0 * height) - (0.0));
[views get:@"label1"].Width = (int)((60 / 100.0 * width));
[views get:@"panellista"].Left = (int)(0.0);
[views get:@"panellista"].Width = (int)((100 / 100.0 * width) - (0.0));
[views get:@"panellista"].Top = (int)(([views get:@"panelencabezadocat"].Top + [views get:@"panelencabezadocat"].Height));
[views get:@"panellista"].Height = (int)((92 / 100.0 * height));
[views get:@"customlistviewcategoria"].Top = (int)((2 / 100.0 * height));
[views get:@"customlistviewcategoria"].Left = (int)((2 / 100.0 * width));
[views get:@"customlistviewcategoria"].Width = (int)((98 / 100.0 * width) - ((2 / 100.0 * width)));
[views get:@"customlistviewcategoria"].Height = (int)((88 / 100.0 * height));

}
@end

File: mobile_app/app_pedidos/Objects/src/com/jq/pedidos/designerscripts/LS_frmcategoria.java
package com.jq.pedidos.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmcategoria{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("panelfondoslidecategorias").vw.setTop((int)(0d));
views.get("panelfondoslidecategorias").vw.setHeight((int)((100d / 100 * height) - (0d)));
views.get("panelfondoslidecategorias").vw.setLeft((int)(0d));
views.get("panelfondoslidecategorias").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("panelencabezadoslidecategoria").vw.setLeft((int)(0d));
views.get("panelencabezadoslidecategoria").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("panelencabezadoslidecategoria").vw.setTop((int)(0d));
views.get("panelencabezadoslidecategoria").vw.setHeight((int)((8d / 100 * height) - (0d)));
views.get("btnbackslidecategoria").vw.setTop((int)(0d));
views.get("btnbackslidecategoria").vw.setHeight((int)((8d / 100 * height) - (0d)));
views.get("btnbackslidecategoria").vw.setLeft((int)(0d));
views.get("btnbackslidecategoria").vw.setWidth((int)((15d / 100 * width) - (0d)));
views.get("label1").vw.setLeft((int)((15d / 100 * width)));
views.get("label1").vw.setTop((int)(0d));
views.get("label1").vw.setHeight((int)((8d / 100 * height) - (0d)));
views.get("label1").vw.setWidth((int)((80d / 100 * width)));
views.get("listviewslidecategorias").vw.setLeft((int)((2d / 100 * width)));
views.get("listviewslidecategorias").vw.setWidth((int)((98d / 100 * width) - ((2d / 100 * width))));
views.get("listviewslidecategorias").vw.setTop((int)((views.get("panelencabezadoslidecategoria").vw.getTop() + views.get("panelencabezadoslidecategoria").vw.getHeight())+(2d / 100 * height)));
views.get("listviewslidecategorias").vw.setHeight((int)((88d / 100 * height)));

}
}